lib/wallaby/node.ex

```diff
@@ -238,10 +238,16 @@ defmodule Wallaby.Node do
   @doc """
   Matches the Node's content with the provided text.
   """
-  @spec has_content?(t, String.t) :: boolean()
+  @spec has_text?(t, String.t) :: boolean()
 
-  def has_content?(%Node{}=node, text) when is_binary(text) do
-    text(node) == text
+  def has_text?(%Node{}=node, text) when is_binary(text) do
+    retry fn ->
+      regex_results = Regex.run(~r/#{text}/, text(node))
+      if regex_results |> is_nil do
+        raise Wallaby.ExpectationNotMet, "Text '#{text}' not found"
+      end
+      true
```

Comment:
The only thing that I'm worried about here is that its non-standard to throw errors from a `?` method. So it might break some people's expectations. I should have thought that through earlier.

Ideally what I'd like to see is that it would retry but only return boolean values. That might require some refactors to the `retry/1` function, but I'm OK with that (it needs to be refactored anyway).